apps/src/templates/ProfessionalLearningApplyBanner.jsx

```diff
@@ -126,83 +25,136 @@ class ProfessionalLearningApplyBanner extends React.Component {
   }
 
   getText() {
-    if (this.props.useSignUpText) {
-      return 'Professional Learning is still on! Applications are open for Middle and High School teachers!';
+    if (this.props.customText) {
+      return this.props.customText;
     } else if (this.props.nominated) {
       return 'CONGRATULATIONS! You’ve been nominated for a scholarship!';
     } else if (this.props.teacherApplicationMode === 'closing-soon') {
-      return 'Applications closing soon! 2021 Professional Learning for Middle and High School teachers';
+      return 'Applications closing soon! 2022 Professional Learning for Middle and High School teachers';
```

Comment:
It'd be lovely if the year auto-incremented or we didn't need to include it so the string was more evergreen, but if engineering work is needed to hide/show the banner anyway not a big deal!

Reply:
Yeah, marketing has also been changing the pages it shows up on, the links it directs to and some of the text slightly, so I think there will always be manual changes necessary